### PR TITLE
fix: unify cached docker

### PR DESCRIPTION
**Pull request by juanmanuel-tirado**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fix minor issues with naming on cached Docker images generated in the CI process.

## Related Tickets & Documents

- Related Issue #
- Closes #

## QA Instructions

_Please replace this line with instructions on how to test your changes._

## [optional] Are there any post deployment tasks we need to perform?

### CODE REVIEW

**Comment by juanmanuel-tirado** [comment]
This will have to wait until #118 is merged.

**Review by jpantos — APPROVED** [review]

**Pull request merged**
